control-plane/agents/src/bin/core/volume/operations.rs

```diff
@@ -569,10 +569,12 @@ impl ResourceReplicas for OperationGuardArc<VolumeSpec> {
 
         assert_ne!(request.replicas, spec_clone.num_replicas);
         if request.replicas > spec_clone.num_replicas {
-            self.increase_volume_replica(registry, state, spec_clone)
+            let _result = self
```

Comment:
why the _result ?

Reply:
Its not needed, I added because it had the guard. But probably its fine.